src/Resources/contao/languages/de/tl_module.php

```diff
@@ -49,3 +50,4 @@
 $GLOBALS['TL_LANG']['merger2']['inherit_all_articles']          = 'Artikel von Elternseite bis zur Wurzel erben';
 $GLOBALS['TL_LANG']['merger2']['inherit_articles_fallback']     = 'Artikel der Seite <em>oder</em> Artikel von Elternseite erben';
 $GLOBALS['TL_LANG']['merger2']['inherit_all_articles_fallback'] = 'Artikel der Seite <em>oder</em> Artikel von Elternseite bis zur Wurzel erben';
+$GLOBALS['TL_LANG']['merger2']['legend_module']                 = 'Module: ';
```

Comment:
I prefer using a placeholder for the Module label here, see comment above. So the value should be `Module: %s`. To translators may change the order of the label/value.

Reply:
changed